packages/yarnpkg-fslib/sources/path.ts

```diff
@@ -138,10 +138,13 @@ function fromPortablePath(p: Path): NativePath {
 
 // Path should look like "N:/berry/scripts/plugin-pack.js"
 // And transform to "/N:/berry/scripts/plugin-pack.js"
-function toPortablePath(p: Path): PortablePath {
+function toPortablePath(p: Path | Buffer): PortablePath {
   if (process.platform !== `win32`)
     return p as PortablePath;
 
+  if (Buffer.isBuffer(p))
+    p = p.toString();
```

Comment:
(Just for reference, this change is redundant with the other change I requested) Having the buffer check after the platform check means that `Buffer`s would still not be supported on unix.

Reply:
Ok